ui/cap-react/src/antd/admin/utils/fieldTypes.js

```diff
@@ -242,7 +242,7 @@ const simple = {
         format: "uri",
       },
       uiSchema: {
-        "ui:widget": "text",
+        "ui:widget": "uri",
```

Comment:
actually can be deleted. No need for it  - should be automatically taken from `format`